api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -344,6 +348,15 @@ public boolean isUserMemberOfGroup(String email, String groupName) {
         });
   }
 
+  @Override
+  public String staticNotebooksConvert(String notebook) {
+    return retryHandler.run(
+        (context) -> {
+          String html = staticNotebooksApiProvider.get().convertNotebook(notebook);
```

Comment:
Can just return

opt: drop the brackets: `context -> staticNotebooksApiProvider.get().convertNotebook(notebook)`